libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -1284,6 +1284,10 @@ typedef unsigned int char32_t;
 #    define _LIBCUDACXX_HAS_NO_BUILTIN_OPERATOR_NEW_DELETE
 #  endif
 
+#  if __check_builtin(__make_integer_seq) && !defined(_LIBCUDACXX_TESTING_FALLBACK_MAKE_INTEGER_SEQUENCE)
```

Comment:
Critical: `__check_builtin` does some unholy things, like prefixing `__` because some keywords are handled differently depending on how it is passed in. So this needs to be:
```suggestion
#  if __check_builtin(make_integer_seq) && !defined(_LIBCUDACXX_TESTING_FALLBACK_MAKE_INTEGER_SEQUENCE)
```